peer/peer.go

```diff
@@ -1057,7 +1075,6 @@ func (p *Peer) handlePingMsg(msg *wire.MsgPing) {
 // ping was not previously sent.
 func (p *Peer) handlePongMsg(msg *wire.MsgPong) {
 	p.statsMtx.Lock()
```

Comment:
Same thing here regarding the lock around the critical section and spacing consistency.  After the comment just before the if statement, please.